### DISCUSSION TITLE
How should we make it easier for non-developers to review PRs?

### DISCUSSION THREAD

**Discussion by willuhmjs** [deep-sea-tactics/website#23] [Q&A]
How should non-developers like @Daniel-Tomov and @Lstriker1268 be able to review PRs? Vercel?

**Comment by tristan-f-r** [comment]
The author of the [promising commit](https://github.com/actions/deploy-pages/commit/71efac92adbe2b78b60e14d420cfcfdf20117a4c) seems to be very active in [actions/deploy-pages](https://github.com/actions/deploy-pages). We might get a release soon! But otherwise, Vercel seems like the best option.

**Comment by CoasterFan5** [comment]
Forced Tristan to use Cloudflare pages.
